### PR TITLE
chore(deps): bump expo to 47.0.3

### PR DESCRIPTION
**Pull request by codeflyer**
Bump expo to 47.0.3.

I created a separated PR to avoid the automerge and allow a last test.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/*
+16
```
Comment (simoneb) [inline .nvmrc:1]: can we please create an issue to remind ourselves that we still want to be able to switch to 18 at some point in the future?

**Comment by simoneb** [comment]
Unfortunately I can't get this to work on expo go, meaning that unless we can prove that it's my problem (meaning that somebody else can run it instead), we can't go ahead with this.

Here's what I did:

- I removed node_modules and all the caches (the .gitignored folders generated by expo)
- upgraded expo-cli, eas-cli and expo go to the latest versions
- connected my phone (android) to my laptop via USB
- ran yarn start

This is what I get consistently:

```
$ yarn start
yarn run v1.22.19
$ expo start --dev-client
Starting project at N:\nearform\optic-expo
Starting Metro Bundler
▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
█ ▄▄▄▄▄ █▄▄██  ▀█▄ ▀▀█▀ █▀█ ▄▄▄▄▄ █
█ █   █ ██  ▀▄  █▀▀▄▄▄ █▀▀█ █   █ █
█ █▄▄▄█ ███ ▀▄▄█▄ ▀▄▄▀▄ ▀██ █▄▄▄█ █
█▄▄▄▄▄▄▄█ █▄█ ▀▄▀▄█ █▄█▄█▄█▄▄▄▄▄▄▄█
█▄▄▀▄▄▀▄▀▀█▄█▄██ ▀ ▄▀ ▀▀▀▄█▄█▀▀██▄█
█▀▀▀▄██▄█▀▄█▄▀▄▄▄▄ █▀▀▄▄▄▄ ▀▄▄█▀  █
██    ▀▄▀▀▄▄ ▀█  ▄▀▀█ ██ █ ▄███▀ ██
█▀▄ ▀ █▄▀▀ ████  ▄█ ▄█▄███▀▄▀█▀ ▄▀█
█▀█▀▄▀ ▄▀█▄▄▄ ▄ ▄█▀█▀▀▀ ▄▄▄   █▀▀ █
█ ███▀▄▄▄▀▀█▄▄ ▄ ▄▄▀▄▄██▄▄█ ▄▄  ▀ █
█▀  █▀ ▄ ▀▀ ▀▀██ ▀▄█▀███▀▄▀█▀▀██ ▄█
█ █▄▄█▀▄ ▀▄█▄▀▄█▄▄ ▀█▀▄  ███▄██  ▄█
█▄▄████▄█▀▄█ ▀█▄ ▄██▄▄▀▄█ ▄▄▄ ▄▀▀▄█
█ ▄▄▄▄▄ ██ ▄▀▄▄▀▄▀█ ▄ ▄█▄ █▄█ ▄▀███
█ █   █ █ ▄▄█  ▄▄█▀   █ ▄▄ ▄ ▄▄█▄██
█ █▄▄▄█ █▀  ▄▀▀▀ █▄█ ▄█ ██▀▀▀▄▀ ▀ █
█▄▄▄▄▄▄▄█▄▄▄████▄█▄▄████▄▄▄██▄█████

› Metro waiting on
exp+optic-expo://expo-development-client/?url=http%3A%2F%2F192.168.1.34%3A8081
› Scan the QR code above with Expo Go (Android) or the Camera app (iOS)

› Press a │ open Android
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu

› Press ? │ show all commands

Logs for your project will appear below. Press Ctrl+C to exit.
Android Bundling complete 4406ms
Android Bundling complete 69ms
 WARN  AsyncStorage has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-async-storage/async-storage' instead of 'react-native'. See https://github.com/react-native-async-storage/async-storage
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.
 ERROR  Error: Cannot find native module 'ExponentCamera'
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called.
```

I have never seen this error before and I couldn't find anything related to "ExponentCamera" online.

@codeflyer I would appreciate if you could:

1. try to run this on your mobile phone via expo go (which is MUCH faster than building for native), see the steps I did above
2. if for any reason you can, ask somebody else to try this

That's because unless we get at least one person to confirm it works on Expo Go, we cannot proceed.

File: .nvmrc (hunk 1 above)
Comment (codeflyer) [reply .nvmrc:1]: Done

**Comment by codeflyer** [comment]
@simoneb I did what you described and it works for me. I had the app working on my phone.

**Review by simoneb — APPROVED** [review]

**Pull request merged**